src/display/annotation_layer.js

```diff
@@ -1440,7 +1440,7 @@ class PushButtonWidgetAnnotationElement extends LinkAnnotationElement {
       container.title = this.data.alternativeText;
     }
 
-    if (this.enableScripting && this.hasJSActions) {
+    if (this.enableScripting && this.hasJSActions && container.lastChild) {
       const linkElement = container.lastChild;
```

Comment:
It seems that we could, ever so slightly, simplify things by getting the element just once:
```suggestion
    const linkElement = container.lastChild;
    if (linkElement && this.enableScripting && this.hasJSActions) {
```